### DISCUSSION TITLE
为什么网络里面只有lo，没有eth0？

### DISCUSSION THREAD

**Discussion by tianyeeT** [lollipopkit/flutter_server_box#236] [Q&A]
RT。
使用的是一键安装脚本安装的，应该在哪里修改配置

**Comment by tianyeeT** [comment]
关联：https://github.com/lollipopkit/flutter_server_box/issues/234
https://github.com/lollipopkit/server_box_monitor/issues/10

**Comment by lollipopkit** [comment]
已在最新版中解决
